src/components/shared/Accordion.js

```diff
@@ -57,6 +57,7 @@ class Accordion extends Component<Props, State> {
         <h2
           className="_header"
           tabIndex="0"
+          onKeyPress={() => this.handleHeaderClick(i)}
```

Comment:
Just to add a little bit of context. The click event is not handling keypress "Enter" events because it's not an actionable (in this case clickable) tag such as <BUTTON>. This is why we explicitly need onKeyPress.

Another suggestion I have is to use key property instead of keyCode (which is deprecated) to check if the value is equal to "Enter" or " ".